apache/datadog_checks/apache/data/conf.yaml.example

```
@@ -1,4 +1,19 @@
 init_config:
+  ## @param skip_proxy - boolean - optional - default: false
+  ## The (optional) skip_proxy parameter bypasses any proxy
+  ## settings enabled.
+  #
+  #  skip_proxy: false
+
+  ## @param proxy - list of string - optional
+  ## Set HTTP or HTTPS proxy for all instances.
+  ## Use the no_proxy list to specify hosts that must bypass proxy
+  #  proxy:
```

```suggestion
  #
  #  proxy:
```

There is a need of one empty commented line